source/object-model/computed-properties.md

```diff
@@ -93,17 +93,16 @@ Person = Ember.Object.extend({
   firstName: null,
   lastName: null,
 
-  fullName: function(key, value, previousValue) {
-    // setter
-    if (arguments.length > 1) {
+  fullName: Ember.computed('firstName', 'lastName', function(key, value, previousValue) {
+    get(key) {
+      return this.get('firstName') + ' ' + this.get('lastName');
```

Comment:
This could be replaced with:

    return `${this.get('firstName')} ${this.get('lastName')}`;

Or maybe

    let firstName = this.get('firstName');
    let lastName = this.get('lastName');

    return `${firstName} ${lastName}`;